STYLEGUIDE.md

```diff
@@ -0,0 +1,101 @@
+# Spyder Documentation Style Guide:
+
+
+## General
+
+Must (not), should (not), recommended, suggested, may, and can generally follow IETF conventions, with some additional differentiation:
+
+- **Must** (not) clauses are hard requirements that must always be observed
+- **Should** (not) clauses can only be disregarded in a particular instance with strong justification, or if it is demonstratively highly impractical to follow for a specific case
+- (Not) **Recommended** and **Preferred** clauses should be followed where practicable; a brief rationale should be provided for not doing so
+- **Suggested** clauses indicate a convention, approach or guideline that may be helpful to follow, but has no binding effect or justification requirements
+- **May** or **Can** clauses designate explicitly permitted guidelines, actions or conventions, often for clarity, that are allowed but have no weight of preference on them (and may even be gently discouraged with a suggested or preferred alternative)
+
```

Comment:
Its just any clause that uses the term, it can be anything. The style guide currently doesn't actually use them thoroughly because I never did the polish pass I planned to long ago before submitting this. Since this section is only normative to the style guide itself rather than to the actual docs, and not consistently employed as of now, I could just remove it.